benchmarks/scripts/benchmark_sota.py

```
@@ -12,216 +12,277 @@
 from collections import defaultdict
+import argparse
+
 try:
     import matplotlib.pyplot as plt
+
     MATPLOTLIB_AVAILABLE = True
 except ImportError:
     MATPLOTLIB_AVAILABLE = False
 
 # Import our modules
 from vecmap import vecmap
```

Ruff (0.12.2)

`vecmap.vecmap` imported but unused

Remove unused import: `vecmap.vecmap`

(F401)